docs/Downloads.md

```diff
@@ -11,16 +10,10 @@ This page describes the various files available for download.
 
 This file is a MySQL database dump for seeding a new instance of the cBioPortal.  It contains all the necessary background data for a properly functioning cBioPortal website, including cancer types, gene, uniprot-mappings, drug and network data.  The file is available for download here:
 
-[cbioportal-seed.sql.gz](http://cbio.mskcc.org/cancergenomics/public-portal/downloads/cbioportal-seed.sql.gz)
+[cbioportal-seed.sql.gz](https://storage.googleapis.com/cbiostorage-1/cbioportal-seed.sql.gz)
```

Comment:
:+1: 
One request, maybe for later: I would like to see this versioned. So maybe at some point we could have a list of links here like 

```
[cbioportal-seed.sql.gz for v1.2.0](https://storage.googleapis.com/cbiostorage-1/seed/1.2.0/cbioportal-seed.sql.gz)
[cbioportal-seed.sql.gz for v1.2.2](https://storage.googleapis.com/cbiostorage-1/seed/1.2.2/cbioportal-seed.sql.gz)
[cbioportal-seed.sql.gz for v1.2.2](https://storage.googleapis.com/cbiostorage-1/seed/1.3.0/cbioportal-seed.sql.gz)
etc
```